app/notebooks/metrics/q_a_generation.ipynb

```diff
@@ -0,0 +1,624 @@
+{
```

Comment:
for reviewers: this file was slightly modified, but git didn't pick up that it was moved, show it shows up as a new file with the old one deleted.